<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>constraintQWindow</class>
 <widget class="QWidget" name="constraintQWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="mastersQBox">
     <property name="title">
      <string>Masters:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="displayOnlySharedQCheck">
        <property name="text">
         <string>Display Only Shared</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="mastersQList">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mastersSelQHLayout">
        <item>
         <widget class="QPushButton" name="addMasterQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add scene selection as master</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="addMasterMOQCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Maintain Offset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delMasterQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove list master</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="replaceMasterQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Replace master list to scene selection</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="mastersSelQSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mastersSelQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Scene select list master</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="constraintQHLayout_1">
        <item>
         <widget class="QPushButton" name="renameAttrQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Rename constraint master attribute based on node connected name</string>
          </property>
          <property name="text">
           <string>Rename Masters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="normalizeWeightsQButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Normalize master attribute weights</string>
          </property>
          <property name="text">
           <string>Normalize Weights</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="relatedCommonQGroup">
     <property name="title">
      <string>Common:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="lockOutputQGroup">
        <property name="title">
         <string>Attrs:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="lockOutputQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Lock Output</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="axisConnectionsQGroup">
        <property name="title">
         <string>Connections:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="QCheckBox" name="connectRxQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>rx</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="connectRzQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>rz</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="connectTzQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>tz</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="connectAllScQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All Sc</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="connectAllXQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All X</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="connectTxQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>tx</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QCheckBox" name="connectSzQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>sz</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="connectRyQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ry</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="connectAllRtQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All Rt</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="connectAllYQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All Y</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="connectSxQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>sx</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="connectAllTrQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All Tr</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="connectAllZQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>All Z</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="connectTyQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ty</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="connectSyQCheck">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>sy</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>displayOnlySharedQCheck</tabstop>
  <tabstop>mastersQList</tabstop>
  <tabstop>mastersSelQButton</tabstop>
  <tabstop>addMasterQButton</tabstop>
  <tabstop>normalizeWeightsQButton</tabstop>
  <tabstop>connectAllTrQCheck</tabstop>
  <tabstop>connectAllRtQCheck</tabstop>
  <tabstop>connectAllScQCheck</tabstop>
  <tabstop>connectAllXQCheck</tabstop>
  <tabstop>connectAllYQCheck</tabstop>
  <tabstop>connectAllZQCheck</tabstop>
  <tabstop>connectTxQCheck</tabstop>
  <tabstop>connectTyQCheck</tabstop>
  <tabstop>connectTzQCheck</tabstop>
  <tabstop>connectRxQCheck</tabstop>
  <tabstop>connectRyQCheck</tabstop>
  <tabstop>connectRzQCheck</tabstop>
  <tabstop>connectSxQCheck</tabstop>
  <tabstop>connectSyQCheck</tabstop>
  <tabstop>connectSzQCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
